packages/prisma-client/prisma/migrations/20230130153140_authorization-tokens-fix/migration.sql

```diff
@@ -0,0 +1,19 @@
+-- CreateEnum
+CREATE TYPE "AuthorizationRelation" AS ENUM ('viewers', 'editors', 'builders');
+
+-- DropTable
+DROP TABLE "AuthorizationTokens";
+
+-- DropEnum
+DROP TYPE "AuthorizationPermit";
+
+-- CreateTable
+CREATE TABLE "AuthorizationToken" (
+    "token" TEXT NOT NULL,
+    "projectId" TEXT NOT NULL,
+    "name" TEXT NOT NULL DEFAULT 'Custom Link',
```

Comment:
"Custom link" should be a UI concern, this is never useful outside of share dialog UI which can always have this, I guess db should be happy with an empty string or something